Mathlib/Analysis/InnerProductSpace/Basic.lean

```diff
@@ -790,14 +790,16 @@ theorem orthonormal_subtype_iff_ite {s : Set E} :
 vectors picks out the coefficient of that vector. -/
 theorem Orthonormal.inner_right_finsupp {v : ι → E} (hv : Orthonormal 𝕜 v) (l : ι →₀ 𝕜) (i : ι) :
     ⟪v i, Finsupp.total ι E 𝕜 v l⟫ = l i := by
-  classical simpa [Finsupp.total_apply, Finsupp.inner_sum, orthonormal_iff_ite.mp hv] using Eq.symm
+  classical!
+  simpa [Finsupp.total_apply, Finsupp.inner_sum, orthonormal_iff_ite.mp hv] using Eq.symm
```

Comment:
Why did this file change? Does the content of the proof change the API here (due to pulling in unwanted `DecidabeEq`s)?